UnityProject/Assets/Scripts/Learning/ProtipObjectTypes/ProtipObjectOnPickupTrait.cs

```diff
@@ -11,8 +11,11 @@ public class ProtipObjectOnPickupTrait : ProtipObject
 		[FormerlySerializedAs("ProtipsForTraits"), SerializeField]
 		private SerializableDictionary<ItemTrait, ProtipSO> protipsForTraits = new SerializableDictionary<ItemTrait, ProtipSO>();
 
-		private void OnEnable()
+		private IEnumerator OnEnable()
 		{
+			//(Max): For some reason, the spawning times will become inconsistent between frames.
+			//To play it say, we intentionally await 2 full seconds until everything has run its course.
+			yield return WaitFor.Seconds(2f);
```

Comment:
is this an initialisation order issue? Because we have that initialsiation interface to make sure things work a set order.

Reply:
not order, timing.

Reply:
This is one of those mind rework things that I don't get why it happens, and Bod caused it and didn't bother fixing one of his issues that he created. This wasn't a hidden thing either; this is an error you always see at the bottom of the editor whenever you start the game when this fix isn't here.

This fix works well, so I see no point in trying to correct it.